projects/2DoBox-Pivot.md

```diff
@@ -0,0 +1,160 @@
+# 2DoBox Pivot
+
+Do you remember that project Ideabox?
+
+We thought they were pretty great, but, I think it's time to pivot those projects over to something we like to call 2DoBox.
+
+You will be inheriting someone's previous IdeaBox and pivoting it over to 2DoBox. Your projects will utilize webpack and it's built in node server. You will also keep accessibility in mind as we will be grading you based on how accessible your application is. You will also implement selenium webdriver to test the DOM.
```

Comment:
`and it's built in node server` - is a little bit confusing. Do you mean the webpack-dev-server here? Might cause readers to conflate the build process with using node as a server for production.